src/cargo/rust-ecosystem.md

```diff
@@ -44,3 +44,7 @@ Key points:
 [editions]: https://doc.rust-lang.org/edition-guide/
 
 </details>
+
+[^rustc]: Please note, it is quite rare to ever use the compiler directly not through `cargo`.
```

Comment:
Please avoid footnotes and instead use the speaker notes as much as possible — it's not immediately clear when you view it on a laptop/desktop, but the material is meant to be shown in a classroom setting. So think of the pages as "slides", not pages :smile: 

The speaker notes can be collapsed or even moved to a popout window. I therefore consider them "free" in terms of space.

Reply:
noted and done